modules/vmfcore/java/jni/com_intel_vmf_FieldDesc.cpp

```diff
@@ -0,0 +1,265 @@
+#include<string>
+#include<vector>
+#include <memory>
+#include <stdexcept>
+#include "vmf/metadatastream.hpp"
+#include "../com_intel_vmf_FieldDesc.h"
+
+using namespace vmf;
+
+static void throwJavaException(JNIEnv *env, const std::exception *e, const char *method)
```

Comment:
I suggest not duplicating this in every cpp, but having a single shared version instead